lib/k6-summary/README.md

```diff
@@ -0,0 +1,5 @@
+# k6-summary
+
+This functions that can be used to generate different end-of-test summary reports when used in the `handleSummary()` callback: [https://k6.io/docs/results-visualization/end-of-test-summary](https://k6.io/docs/results-visualization/end-of-test-summary/).
```

Comment:
I think we should link to https://grafana.com/docs/k6/latest/results-output/end-of-test/, unless there's a separate issue for updating links in this repo

Reply:
Yeah, it's a good point :+1:

Reply:
```suggestion
This functions that can be used to generate different end-of-test summary reports when used in the `handleSummary()` callback: [https://grafana.com/docs/k6/latest/results-output/end-of-test/](https://grafana.com/docs/k6/latest/results-output/end-of-test/).
```